tests/test_workers/test_tasks/test_build.py

```diff
@@ -455,6 +457,7 @@ def test_handle_add_request(
 ):
     arches = {'amd64', 's390x'}
     binary_image_config = {'prod': {'v4.5': 'some_image'}}
+    mock_iidc.return_value = False
```

Comment:
It would not make any change to tested behavior. 
We are not checking logged messages. The only change in behavior is logged WARNING when it is set to `True`.
There fore I did not set it anywhere.

Reply:
ok, thank for reply :)